app/views/support_interface/validation_errors/vendor_api/index.html.erb

```diff
@@ -0,0 +1,59 @@
+<% content_for :title, 'Vendor API validation errors' %>
+
+<% content_for :before_content do %>
+  <%= breadcrumbs({
+    'Performance': support_interface_performance_path,
+    'Validation errors': support_interface_validation_errors_path,
+    'Vendor API': nil,
+  }) %>
+<% end %>
+
+<p class="govuk-body"><%= govuk_link_to 'Validation error summary', support_interface_validation_errors_vendor_api_summary_path %></p>
+
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header govuk-!-width-three-quarters">Attribute error</th>
+      <th class="govuk-table__header govuk-table__header--numeric govuk-!-width-one-quarter">Error count</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+  <% @grouped_column_error_counts.each do |(request_path, error, message), count| %>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
+          <span class="govuk-!-font-weight-regular"><%= govuk_link_to request_path, support_interface_validation_errors_vendor_api_search_path(request_path: request_path) %>:</span>
+          <%= govuk_link_to error.humanize, support_interface_validation_errors_vendor_api_search_path(request_path: request_path, attribute: error) %>
+        </h2>
+        <p class="govuk-body govuk-!-margin-bottom-0"><%= message %></p>
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric">
+        <%= count %>
+      </td>
+    </tr>
+  <% end %>
+  </tbody>
+</table>
+
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header govuk-!-width-three-quarters">Requests</th>
+      <th class="govuk-table__header govuk-table__header--numeric govuk-!-width-one-quarter">Error count</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+  <% @grouped_counts.each do |request, count| %>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
+          <%= govuk_link_to request, support_interface_validation_errors_vendor_api_search_path(request_path: request) %>
+        </h2>
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric">
+        <%= count %>
+      </td>
+    </tr>
+  <% end %>
```

Comment:
Excluding this bit of code, the rest can potentially be extracted to an ErrorsListComponent that can be reshared between candidate and vendor API. A content block can be specified in the component (similar to [here](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/components/collection_select_component.html.erb#L15) to enable us to leave this code in the view.

What do you think? I'm not sure if i missed anything else that makes that more complicated.

Reply:
done in 1dda5794495a138725549e3e11ff2fbcf7951e14